src/app/app.module.ts

```diff
@@ -30,7 +31,9 @@ import { AddressComponent } from './address/address.component';
     BrowserModule,
     IonicModule.forRoot(),
     AppRoutingModule,
-    SharedModule
+    SharedModule,
+    HomePageModule
```

Comment:
we don't need HomePageModule in the app.module.ts if we are loading the HomePageModule lazily i.e using loadChildren in app.routing.module